src/ScottPlot5/ScottPlot5/Primitives/RenderDetails.cs

```diff
@@ -99,6 +99,99 @@ public RenderDetails(RenderPack rp, (string, TimeSpan)[] actionTimes, RenderDeta
         SizeChanged = !DataRect.Equals(lastRender.DataRect);
     }
 
+    /// <summary>
+    /// Search the <see cref="AxisLimitsByAxis"/> for the specified <paramref name="axis"/> and calculate the pixels per unit
+    /// </summary>
+    /// <param name="axis">the X-Axis to search for</param>
+    /// <param name="value">the pxPerUnitX</param>
+    /// <returns>
+    /// <see langword="true"/> if the axis was in the collection and a result was calculated. Otherwise <see langword="false"/>
+    /// <br/> When false, the <paramref name="value"/> will be set to <see cref="PxPerUnitX"/>
+    /// </returns>
+    public bool TryGetPixelPerUnitX(IXAxis? axis, out double value)
+    {
+        // To-DO : Convert to 'GetPixelPerUnitX' ? Throw if axis not found ?
```

Comment:
I guess this TODO doesn't make any sense anymore?

Reply:
I left that to do in there to get your opinion on the matter. Nothing I wrote uses the bin result, so it would be easy to Convert it. 

I also noticed in your PR B that you had the throw if an unknown axis  was supplied. I think I would rather just get the default, but throwing also makes sense in that scenario, So I wanted a second opinion on the matter

Reply:
I like the Try option better. It is more flexible, and shifts the decision making to the user of the method. 
On the other hand, you don't check the result yourself, and others won't either

Reply:
Well, this was also running under the assumption of 'best match'. Best case it finds the axis, worst case it falls back to default if an axis that doesn't exist is supplied (bringing it back to original functionality)

So I'm torn on this as well. It doesnt hurt like this, but the result value cannot be used directly either.

Reply:
Keeping it as try for reasons you described. The way its currently being used will at worst not find a value on the plot, or find one thats not quite correct. No worse than original functionality.